tests/main/interfaces-desktop-document-portal/task.yaml

```diff
@@ -0,0 +1,60 @@
+summary: The document portal is mounted by snaps using the desktop interface
+details: |
+    The document portal is a component of xdg-desktop-portal that
+    provides a way to share files with a confined application in a
+    controlled fashion.  To provide proper security, a subtree of the
+    portal needs to be mounted over $XDG_RUNTIME_DIR/doc inside the
+    sandbox.
+systems: [-ubuntu-core-*]
```

Comment:
This also proves my point about that this interface feature should be just classic.